tests/MoBi.Tests/Presentation/Tasks/InteractionTasksForRootApplicationBuilderSpecs.cs

```diff
@@ -52,7 +50,7 @@ protected override void Because()
       [Observation]
       public void should_not_return_builder_name_as_forbidden()
       {
-         _result.ShouldOnlyContain(_builderName);
+         _result.ShouldNotContain(_builderName);
```

Comment:
Trying to infer this test from the name and content. It feels like the original test was just using the wrong assertion.